integration-job/config.yaml

```diff
@@ -0,0 +1,19 @@
+presubmits:
+  kyma-project/kyma:
+  - name: kyma-integration
+    trigger: "(?m)^/test kyma-integration"
+    rerun_command: "/test kyma-integration"
+    context: kyma-integration
+    always_run: true
+    skip_report: false
+    agent: kubernetes
+    spec:
```

Comment:
currently, I am able to run this job many times concurrently for the same PR. I am afraid of a malicious user who can easily create hundreds of VMs. How do we want to prevent that?